packages/server/src/routes.ts

```diff
@@ -62,24 +67,39 @@ export const createRouter = (ctx: AppContext): express.Router => {
   })
 
   // Get operation log for a DID
-  router.get('/log/:did', async function (req, res) {
+  router.get('/:did/log', async function (req, res) {
     const { did } = req.params
     const log = await ctx.db.opsForDid(did)
     if (log.length === 0) {
       throw new ServerError(404, `DID not registered: ${did}`)
     }
-    res.json({ log })
+    res.json(log)
+  })
+
+  // Get operation log for a DID
+  router.get('/:did/auditable', async function (req, res) {
+    const { did } = req.params
+    const ops = await ctx.db.indexedOpsForDid(did, true)
+    if (ops.length === 0) {
+      throw new ServerError(404, `DID not registered: ${did}`)
+    }
+    const log = ops.map((op) => ({
+      ...op,
+      cid: op.cid.toString(),
+      createdAt: op.createdAt.toISOString(),
+    }))
+
+    res.json(log)
   })
 
   // Get the most recent operation in the log for a DID
-  router.get('/last/:did', async function (req, res) {
+  router.get('/:did/last', async function (req, res) {
```

Comment:
In a similar line of thought re: `/:did/log/audit`, I could also imagine calling this `/:did/log/last`.  Really not too hung-up on it, really just one final poke to make sure we like the naming conventions here!

Reply:
actually you're right i'll do those 👍